demos/hazard/NonparametricSourcesClassicalPSHA/job.ini

```diff
@@ -0,0 +1,57 @@
+[general]
+
+description = Classical PSHA with Non-parametric sources
+calculation_mode = classical
+random_seed = 23
+
+
+[geometry]
+
+region = -78.0 4.0, -76.0 4.0, -76.0 2.0, -78.0 2.0
+region_grid_spacing = 20.0
+
+
+[logic_tree]
+
+number_of_logic_tree_samples = 0
+
+[erf]
+# km
+rupture_mesh_spacing = 5.0
+width_of_mfd_bin = 0.1
+
+
+[site_params]
+
+reference_vs30_type = measured
+reference_vs30_value = 800.0
+reference_depth_to_2pt5km_per_sec = 5.0
+reference_depth_to_1pt0km_per_sec = 40.0
+
+[calculation]
+
+source_model_logic_tree_file = source_model_logic_tree.xml
+gsim_logic_tree_file = gmpe_logic_tree.xml
+
+# years
+investigation_time = 1.0
+optimize_same_id_sources = true
+intensity_measure_types_and_levels = {"PGA":      logscale(0.001,2.0, 30),
+                                      "SA(0.2)":  logscale(0.001,4.0, 30),
+                                      "SA(0.5)":  logscale(0.001,2.0, 30),
+                                      "SA(1.0)":  logscale(0.001,2.0, 30),
+                                      "SA(2.0)":  logscale(0.001,1.0, 30)}
+
+truncation_level = 3
+# integration distance in km (J&B distance)
+maximum_distance = {'Subduction IntraSlab': 300.,'default': 300.}
+
+
+[output]
+
+export_dir = /tmp
+mean_hazard_curves = True
+quantile_hazard_curves =
```

Comment:
Is this an expected format?

Reply:
You can remove `mean_hazard_curves = True quantile_hazard_curves =` since it is the default.